INSTALL.md

```diff
@@ -11,9 +11,24 @@
 
     Examples: `/home/myuser/www/web/` or `/var/sites/<project-name>/web/`
 
-## Clean install
+## Prerequisite
 
-1. You can get eZ Platform using composer with the following commands:
+  These instructions assumes you have strong technical knowledge and have already installed PHP, web server & a database server with a corresponding clean database needed for this software.
+  For further information see: https://doc.ez.no/display/EZP/Requirements
+
+## Install
+
+1. **Get eZ Platform**:
+
+       A. **Archive** (tar/zip)
+
+       - Extract the eZ Platform 2015.01 or higher archive to a directory (herby refered to as `<root-dir>`).
+       - Modify ezpublish/config/parameters.yml to reflect your database name/server
```

Comment:
Will the tarball *contain* parameters.yml ? The alternative is to instruct to run composer update or composer run-script post-install-cmd so that people get the same experience than those doing create-project.